activitysim/examples/placeholder_psrc/configs/stop_frequency_annotate_tours_preprocessor.csv

```diff
@@ -45,3 +45,6 @@ AccesibilityADestination off-peak transit,_dest_trOpRetail,"reindex(accessibilit
 AccesibilityAtDestination if transit,pracc,"pracc.where(~tour_mode_is_transit, _dest_trPkRetail.where(_tour_starts_in_peak, _dest_trOpRetail))"
 AccesibilityAtDestination if non_motorized,pracc,"pracc.where(~tour_mode_is_non_motorized, reindex(accessibility.nmRetail, df.destination))"
 ,destination_area_type,"reindex(land_use.area_type, df.destination)"
+,is_mixed,df.composition=='mixed'
```

Comment:
general comment: the changes we are making in all these model spec and preprocessor files, are they *required* or just helpful?  We don't want to have required changes that injure backwards compatibility.

Reply:
These are required in sharrow mode. It is related to [issue 766](https://github.com/ActivitySim/activitysim/issues/766). I noticed that if a pandas categorical variable is compared to a string, sharrow seems to evaluate it wrong.

Reply:
ok, I have added 766 to the Phase 9 priority list.